Formula/handbrake.rb

```diff
@@ -26,12 +26,23 @@ class Handbrake < Formula
   depends_on "yasm" => :build
 
   uses_from_macos "m4" => :build
+  uses_from_macos "bzip2"
   uses_from_macos "libxml2"
+  uses_from_macos "zlib"
 
   on_linux do
     depends_on "jansson"
+    depends_on "jpeg-turbo"
+    depends_on "lame"
+    depends_on "libass"
+    depends_on "libvorbis"
+    depends_on "libvpx"
```

Comment:
`libvpx` is statically linked with either formula or bundled copy.
Can wait for #85088, which should add shared library.

Reply:
Rebased, and confirmed linkage using objdump:
```
Dynamic Section:
  NEEDED       libass.so.9
  NEEDED       libmp3lame.so.0
  NEEDED       libvpx.so.6
```